src/components/AnnuityResult/AnnuityResult.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+
+import NumberFormat from "react-number-format";
+
+import classes from "./AnnuityResult.module.scss";
+
+export function AnnuityResult(props: any) {
+  return (
+    <div className={classes.resultsContainer}>
+      <div>
+        Ежемесячный платёж:{" "}
```

Comment:
As you could see, some of the UI labels in the existing code are used from display-names constants. This is for supporting multiple languages in future. So it'd be better to follow this.
(yeah I know that we already broke this rule in some places, but we were trying to make it work asap😊)